Makefile

```diff
@@ -79,7 +79,7 @@ OPERATOR_IMAGE_NAME ?= common-service-operator
 # Current Operator bundle image name
 BUNDLE_IMAGE_NAME ?= dev-common-service-operator-bundle
 
-CHANNELS := v3.20
+CHANNELS := v3.20.1
```

Comment:
Hi @qpdpQ 

We don't change the channel for cs-operator.

It will stay at `v3.20` channel for all `v3.20.x` versions